themes/app/templates/Includes/HomePageAccom.ss

```diff
@@ -0,0 +1,25 @@
+<div class="accommodation container">
+    <div class="row justify-content-center">
+        <h2 class="accommodation__heading">$AccomHeading</h2>
```

Comment:
![screen shot 2018-12-06 at 12 02 26 pm](https://user-images.githubusercontent.com/27745093/49549858-dddfd800-f94e-11e8-9b56-de0f9d120ba3.png)

Might need to increase spacing under the heading :)